sympy/core/expr.py

```diff
@@ -23,6 +24,10 @@ class Expr(Basic, EvalfMixin):
     used only for argument storage and expression manipulation, i.e.
     pattern matching, substitutions, etc).
 
+    Should us _eval_is_X (_eval_is_gt/_eval_is_lt _eval_is_le) if we want to override
```

Comment:
us -> use and is ge left off on purpose? perhaps parenthetical should read "(where X is gt, lt, ge, or le)"

Reply:
Ah, yes you are correct thank you.